### PR TITLE
chore(release): version 1.15.0

### PR DESCRIPTION
**Pull request by alacambre-yseop**

### CODE REVIEW

**Comment by mminot-yseop** [comment]
I have no recollection of reviewing those features and fixes xD

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+## 1.15.0 (2021-09-17)
+
+-   Greatly improved the performances of current parser, as well as the kind of expressions it can parse such as
+    -   `modify`
+    -   `whatever`
+    -   `exists`
+    -   `noExists`
+    -   static instances of `Condition`
+    -   `as`
+    -   etc…
```
Comment (mminot-yseop) [inline CHANGELOG.md:12]: :triggered:

```suggestion
    -   etc.
```

# or

```suggestion
    -   … and more.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+-   [BREAKING CHANGE] Semicolon is now mandatory where it was just optional before.
```
Comment (mminot-yseop) [inline CHANGELOG.md:13]:
```suggestion
-   [BREAKING CHANGE] Semicolons are now mandatory where they were just optional before.
```

File: client/resources/kb/_technical/toFormat/otherDocumentFormat.dcl [hunk 3]
```diff
@@ -2,13 +2,13 @@ function myFunction2(World world, Person me, Person her)
...
+    you     =new(Person, firstName, "Peter", lastName, "Pan");
     me.friend:=     you;
-    world.execute(me)
+    world.execute(me);
     if(world.find(you)==true    ||world.find(her) !=false){
-        logInfo("Found somebody.")
+        logInfo("Found somebody.");
     }else{
-        logWarning("Find nobody")
+        logWarning("Find nobody");
```
Comment (mminot-yseop) [inline client/resources/kb/_technical/toFormat/otherDocumentFormat.dcl:11]: xD You’re the first victim of the breaking change.

**Review by mminot-yseop — COMMENTED** [review]
Flash-review.

File: client/resources/kb/_technical/toFormat/otherDocumentFormat.dcl (hunk 3 above)
Comment (alacambre-yseop) [reply client/resources/kb/_technical/toFormat/otherDocumentFormat.dcl:11]: Yeah, it was quite painful.
For the tests it is mandatory to have everything updated, while in a project it's still OK and you can stick to an older version if you prefer.

**Review by mminot-yseop — APPROVED** [review]
OK for me if @erobert-yseop tests.
If needed, ping me and I’ll try to run it but not sure what I should do regarding the new stuff.

**Comment by alacambre-yseop** [comment]
The new stuff have already been tested.

**Review by erobert-yseop — APPROVED** [review]
The tests are OK :+1:

**Pull request merged**